contentcuration/contentcuration/settings.py

```diff
@@ -305,3 +305,7 @@
 AWS_AUTO_CREATE_BUCKET = True
 AWS_S3_FILE_OVERWRITE = True
 AWS_S3_BUCKET_AUTH = False
+
+# GOOGLE DRIVE SETTINGS
+GOOGLE_AUTH_JSON = "credentials/client_secret.json"
+GOOGLE_STORAGE_REQUEST_SHEET = "1uC1nsJPx_5g6pQT6ay0qciUVya0zUFJ8wIwbsTEh60Y"
```

Comment:
Should we change the actual sheet in production?

Reply:
That's the one we're using, but dev_settings does have a different sheet